src/create/components/ScheduleTab.tsx

```diff
@@ -0,0 +1,42 @@
+// 일정 탭 컴포넌트
+import React from "react";
+import { TabType } from "./Create";
+
+interface ScheduleTab {
+  currentTab: string;
+  onClickTab: (tab: TabType) => void;
+}
+
+const ScheduleTab = ({ currentTab, onClickTab }: ScheduleTab) => {
+  return (
+    <>
+      <div className="text-zinc-800 text-[32px] font-bold capitalize">
+        모든 일정
+      </div>
+      <div className="h-[30px] relative mt-9 flex gap-x-10">
+        {tabItems.map((tab, i) => (
+          <div
+            key={`schedule Tab-${i}`}
+            className={`cursor-pointer ${
+              currentTab === tab.title
+                ? "border-b-2 border-pink-400 text-zinc-800 font-bold"
+                : "text-zinc-400 font-medium"
+            }`}
+            onClick={() => onClickTab(tab.title as TabType)}
+          >
+            {tab.title}
+          </div>
+        ))}
+      </div>
+    </>
+  );
+};
+
+export default ScheduleTab;
+
+const tabItems = [
```

Comment:
```suggestion
const tabItems: Record<"title", TabType>[] = [
```

[Lv2] 이렇게 하면 25번줄에 타입 단언을 쓰지않아도 사용할수있습니다.
`as Type` 형식의 타입단언은 가능하다면 최소한으로 사용하는게 디버그할때 좋습니다!

Reply:
확인했습니다! :)